### ISSUE TITLE
PDF ist not loaded

### ISSUE THREAD

**Issue by kjancke** [Audiveris/audiveris#280]
I'm getting the following error when I try to load a PDF:
[Predlue_PDF-0001] 1 sheet in /home/karsten/Documents/Development/audivers/Predlue_PDF-0001.pdf
[Predlue_PDF-0001] Error in performing [LOAD, BINARY] java.util.concurrent.ExecutionException: de.intarsys.pdf.cos.COSRuntimeException: de.intarsys.pdf.parser.COSLoadError: EI expected at character index 5044609

Any idea what's wrong?
[Predlue_PDF-0001.pdf](https://github.com/Audiveris/audiveris/files/3145810/Predlue_PDF-0001.pdf)

**Comment by hbitteur** [comment]
According to the log, this error occurred in JPod software, an external library to which Audiveris delegates the loading of PDF files. We have no control on its internals.

Googling on "de.intarsys.pdf.parser.COSLoadError: EI expected at character index" leads for example to [this message](https://sourceforge.net/p/jpodlib/discussion/727492/thread/409c57c7/) which refers to some abnormal sequence within the PDF stream.
Some PDF loaders may cope with it, some others won't.

I'd suggest to discard this PDF file and replace it by some other image file (perhaps another PDF file, but produced by a different scan software).

**Comment by hbitteur** [comment]
@kjancke 
Here is a workaround.

Open your PDF file with a PDF reader (I used Foxit reader), and then print the image to a virtual printer (I selected "Foxit Reader PDF Printer").
This gives you a new PDF file, containing the same image, but encoded by a different software (Foxit Reader this time).
And this new PDF file is correct enough to be read by Audiveris/JPod.

Granted, it's just a workaround, but it works :-)
